docs/source/reference/core/configuration.rst

```diff
@@ -36,10 +36,10 @@ Note that the default values are set in the global config.
    See :ref:`debug` for more information of the debug mode.
    The default value is given by ``CHAINER_DEBUG`` environment variable (set to 0 or 1) if available, otherwise uses ``False``.
 ``chainer.config.enable_backprop``
-   Flag to enable backpropagation support.
-   If it is ``True``, :class:`Function` makes a computational graph of :class:`Variable` for back-propagation.
-   Otherwise, it does not make a computational graph.
-   So a user cannot call :func:`~chainer.Variable.backward` method to results of the function.
+   Flag to enable back-propagation support.
```

Comment:
"backpropagation" is usually spelled without a hypen. It's better to remove it.

Reply:
Thanks, now fixed!